release notes/v0.45.0.md

```diff
@@ -0,0 +1,169 @@
+
+### UX improvements and enhancements
+
+- [#3015](https://github.com/grafana/k6/pull/3015) Official arm64 docker images. Thanks to @nrxr! 🙇
+- [#3099](https://github.com/grafana/k6/pull/3099) replace "breached" with "crossed" in logs around thresholds. Thanks to @MattDodsonEnglish 🙇.
+- [#3102](https://github.com/grafana/k6/pull/3102) Better error message when SharedArray constructor is provided with an async function. This is not supported, but the original message wasn't very clear.
+- [#3089](https://github.com/grafana/k6/pull/3089) Add SBOM reports to k6 releases. Thanks to @SadFaceSmith 🙇.
+
+## Bug fixes
+
+* [#3058](https://github.com/grafana/k6/pull/3058) fix repetitive `the`. Thank you, @cuishuang 🙇.
+
+## Maintenance and internal improvements
+
+- [#2091](https://github.com/grafana/k6/pull/2991) Refactor JS modules system so that is usable in tests. Which allowed the enablement of the tc39 tests for modules [#3040](https://github.com/grafana/k6/pull/3040).
+- [#3025](https://github.com/grafana/k6/pull/3025) Internally stop referring to afero and use an internal package to do all file system interaction. That package still uses afero.
+- [#3036](https://github.com/grafana/k6/pull/3036) and [#3053](https://github.com/grafana/k6/pull/3053) Add options to `scenarios` for usage by browser module. 
+- [#3064](https://github.com/grafana/k6/pull/3064), [#3070](https://github.com/grafana/k6/pull/3070),  [#3075](https://github.com/grafana/k6/pull/3075) and [#3106](https://github.com/grafana/k6/pull/3106) Go dependancies updates. 
+- [#3067](https://github.com/grafana/k6/pull/3067) Add method to retrive all registered metrics.
+- [#3068](https://github.com/grafana/k6/pull/3068) Add metric Sink constructor.
+- [#3078](https://github.com/grafana/k6/pull/3078) Pinning to alpine 3.17 in docker builder image. Thank you, @arukiidou 🙇.
+- [#3086](https://github.com/grafana/k6/pull/3086) Fix downlonad `.golangci.yml` for PRs from forks. 
+- [#3088](https://github.com/grafana/k6/pull/3088) Make TestDNSResolver less flaky. 
+- [#3094](https://github.com/grafana/k6/pull/3094) Fix example from the run command. Thanks to @rocktwotj 🙇.
+- [#3095](https://github.com/grafana/k6/pull/3095) Maintenance update of .`golangci.yml`.
+- [#3103](https://github.com/grafana/k6/pull/3103) Fix lint and logical issues in `k6/data` module tests.
+- [#3045](https://github.com/grafana/k6/pull/3045), [#3049](https://github.com/grafana/k6/pull/3049), [#3073](https://github.com/grafana/k6/pull/3073) and [#3044](https://github.com/grafana/k6/pull/3044) New issues are now assigned to maintaners. This is to better split the triaging of new issues between maitainers. Also, both that and new PRs assignement actions are now not using external actions.
+## Roadmap
+
+There is now a [public Roadmap document](https://github.com/grafana/k6/blob/master/ROADMAP.md) in the main repo added in [#3028](https://github.com/grafana/k6/pull/3028)
```

Comment:
I mean we are hitting the short-term goals I guess :shrug: 

Don't know if we should update them now or keep it for a little while longer :shrug: 

I guess we will need to update them each release/planning phase or something similar - so we can do it during this cooldown period

Reply:
>I mean we are hitting the short-term goals I guess 

That's for sure and great :sweat_smile: 

My concern is that some people will see this doc the first time (since the release notes are quite a strong channel of promotion), and they could be confused if they see obsolete things.

So yeah this cooldown sounds great.

Reply:
What is the plan here? Are we going to open a PR for updating it?

Reply:
I've created a PR for it https://github.com/grafana/k6/pull/3119